### PR TITLE
Clarify variable name

### PR DESCRIPTION
**Pull request by kevinebaugh**
@naitoh what do you think?

### CODE REVIEW

File: lib/rexml/source.rb [hunk 1]
```diff
@@ -87,7 +87,7 @@ def initialize(arg, encoding=nil)
         detect_encoding
       end
       @line = 0
-      @term_encord = {}
+      @encoded_term = {}
```
Comment (olleolleolle) [inline lib/rexml/source.rb:90]: Minor: How about making it a plural?

```suggestion
      @encoded_terms = {}
```

File: lib/rexml/source.rb [hunk 2]
```diff
@@ -246,7 +246,7 @@ def read(term = nil, min_bytes = 1)
 
     def read_until(term)
       pattern = Private::PRE_DEFINED_TERM_PATTERNS[term] || /#{Regexp.escape(term)}/
-      term = @term_encord[term] ||= encode(term)
+      term = @encoded_term[term] ||= encode(term)
```
Comment (olleolleolle) [inline lib/rexml/source.rb:249]:
```suggestion
      term = @encoded_terms[term] ||= encode(term)
```

**Review by olleolleolle — COMMENTED** [review]
How about making the name a plural?

**Comment by naitoh** [comment]
> How about making the name a plural?

Me, too.
`@encoded_terms` looks good.

**Comment by kevinebaugh** [comment]
> > How about making the name a plural?
> 
> Me, too. `@encoded_terms` looks good.

👍 29b2bec6bc4903f887e45c05fc4074eec2866d24

**Comment by kou** [comment]
Thanks.

**Pull request merged**